src/commands/shared.ts

```diff
@@ -108,6 +105,10 @@ export const withAwsKmsSignerOption = (yargs: Argv): Argv =>
       type: "string",
       description:
         "AWS KMS key id. Example: arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab",
+    })
+    .option("use-env", {
```

Comment:
dont like the name, it's not explicit that it's for was km case

Reply:
instead of a flag, maybe we can add a type, so that's it's easier to know which case we are in.

To avoid breaking change the type will still be optional, but we can consider a breaking change in the future

something like --signer-type="kms"

Reply:
since we no longer need to branch (below comment) we can remove this arg entirely